server/webapp/WEB-INF/rails.new/app/views/preferences/notifications.html.erb

```diff
@@ -0,0 +1,27 @@
+<header class="page-header no-buttons">
+  <h1>Preferences</h1>
+</header>
+
+<ul class="sub-tab-navigation">
+  <li class="current"><a>Notifications</a></li>
+  <li><%= link_to "OAuth Access Token", oauth_engine.user_tokens_index_path %></li>
+</ul>
+<div class="sub-tab-content">
+
+  <% unless go_config_service.is_smtp_enabled %>
```

Comment:
We prefer to use java convention when calling java methods. Mostly because the IDE highlights things when renaming java methods. Otherwise, one has to remember to look for usages of `camel_case` and `camel_case?` and `camel_case=` all over the place.

So this should be written as `go_config_service.isSmtpEnabled`.